modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +62,386 @@ and device ID 0x1015
-$ oc create -f sriov-cni.yaml
+  dpConfigVersion: d41d8cd98f00b204e9800998ecf8427e
+status:
+  interfaces:
+  - deviceID: "1017"
+    driver: mlx5_core
+    mtu: 1500
+    name: ens785f0
+    pciAddress: "0000:18:00.0"
+    totalvfs: 8
+    vendor: 15b3
+  - deviceID: "1017"
+    driver: mlx5_core
+    mtu: 1500
+    name: ens785f1
+    pciAddress: "0000:18:00.1"
+    totalvfs: 8
+    vendor: 15b3
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens817f0
+    pciAddress: 0000:81:00.0
+    totalvfs: 64
+    vendor: "8086"
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens817f1
+    pciAddress: 0000:81:00.1
+    totalvfs: 64
+    vendor: "8086"
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens803f0
+    pciAddress: 0000:86:00.0
+    totalvfs: 64
+    vendor: "8086"
+  syncStatus: Succeeded
 ----
 
-== Configuring additional interfaces using SR-IOV
+== Configuring SR-IOV network devices
+
+The SR-IOV Network Operator introduces `SriovNetworkNodePolicy` Custom Resource
+Definition (CRD) to define the SR-IOV network device and the configuration of
+SR-IOV device plug-in
+
+You should never have to modify this CRD. To make changes to your deployment,
+create and modify a specific Custom Resource (CR). Instructions for creating or 
+modifying a CR are provided in this documentation as appropriate.
+
+[NOTE]
+=====
+To make the configuration change take effect, creating or modifying the Custom
+Resource of `SriovNetworkNodePolicy` may trigger operator to drain the nodes,
+and in some cases reboot the nodes.
+
+The whole process may take several minutes. All configuration changes shall
+have been applied until all the pods in `sriov-network-operator` namespace are
+in `Running` status.
+=====
+
+The following is an example of a typical Custom Resource for
+`SriovNetworkNodePolicy`.
 
-. Create a YAML file for the Custom Resource (CR) with SR-IOV configuration. The
-`name` field in the following CR  has the value `sriov-conf`.
-+
 [source,yaml]
 ----
-apiVersion: "k8s.cni.cncf.io/v1"
-kind: NetworkAttachmentDefinition
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
 metadata:
-  name: sriov-conf
-  annotations:
-    k8s.v1.cni.cncf.io/resourceName: openshift.io/sriov <1>
+  name: policy-example <1>
+  namespace: sriov-network-operator <2>
 spec:
-  config: '{
-      "type": "sriov", <2>
-      "name": "sriov-conf",
-      "ipam": {
-        "type": "host-local",
-        "subnet": "10.56.217.0/24",
-        "routes": [{
-          "dst": "0.0.0.0/0"
-        }],
-        "gateway": "10.56.217.1"
-      }
-    }'
+  resourceName: sriov <3>
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true" <4>
+  priority: 99 <5>
+  mtu: 9000 <6>
+  numVfs: 16 <7>
+  nicSelector: <8>
+    vendor: "15b3" <8>
+    deviceID: "" <9>
+    pfName: ["eno3", "eno4"] <10>
+    rootDevices: ["0000:02:00.0", "0000:02:00.1"] <11>
+  deviceType: netdevice <12>
+  isRdma: false <13>
```

Comment:
I Did not see `isRdma` option in the SriovNetworkNodePolicy CR in my env.

Reply:
`isRdma` is empty if not set : https://github.com/openshift/sriov-network-operator/blob/master/pkg/apis/sriovnetwork/v1/sriovnetworknodepolicy_types.go#L36
If you take a look at the device plugin configMap generated by operator, it shall be configured as `False` if not set.

Reply:
@zhaozhanqi  is this still an issue in your test?

Reply:
@zshi-redhat yes, as you said, it's the false by default.
but I think we should add this one to template:
```
apiVersion: sriovnetwork.openshift.io/v1
kind: SriovNetworkNodePolicy
metadata:
  name: policy-1
  namespace: sriov-network-operator
spec:
  resourceName: intelnics
  nodeSelector:
    feature.node.kubernetes.io/sriov-capable: 'true'
  priority: 99
  mtu: 9000
  numVfs: 6
  nicSelector:
    vendor: '8086'
    rootDevices:
      - '0000:01:00.1'
    pfNames:
      - eth1
  deviceType: vfio-pci
```